src/styles/app.styl

```diff
@@ -470,6 +470,7 @@ footer
     display flex
     justify-content center
     align-items stretch
+    margin-top 27px
```

Comment:
oh right. Missed this one. The thing is there's no Cozy-UI on this repo, so we can't use functions like em()… Sooooo until then let's use em by doing do the math ourselves